CHANGELOG.md

```diff
@@ -1,7 +1,28 @@
 # Change Log
+
 All notable changes to this project will be documented in this file.
 See the [change log guidelines](http://keepachangelog.com/) for information on how to structure the file.
 
+## [5.1.0]
+
+### Added
+
+- Simplify the setup process with automated creation of deegree configurations and capabilities for WFS/WMS services.
```

Comment:
I can't find anything on this, what commits does this refer to?

Reply:
Here it is
https://github.com/halestudio/hale-cli/commit/7ca5a714bf96f734c25764e9adfd9472d6e651be
https://wetransform.atlassian.net/browse/SVC-516
I have just taken the title of the issue.
Should we keep it?

Reply:
This is covered by "Added support for specifying filters when loading data".
It was work done in the context of the ticket, but the ticket itself is not a task on hale-cli.

Reply:
Thank you. I have remove it.